ui/src/app/views/account-creation/component.spec.tsx

```diff
@@ -86,6 +86,17 @@ it('should handle username validity contains special chars', () => {
 });
 
 it('should handle username validity long but has mismatch at end', () => {
+  const wrapper = component();
+  expect(wrapper.exists('#username')).toBeTruthy();
+  expect(wrapper.exists('#usernameError')).toBeFalsy();
+  // if username is long (not too long) but has a mismatch at end
+  let testInput = fp.repeat(50, 'abc');
+  testInput = testInput + ' abc';
+  wrapper.find('input#username').simulate('change', {target: {value: testInput}});
+  expect(wrapper.exists('#usernameError')).toBeTruthy();
+});
+
+it('should handle username validity length less than 3 characters', () => {
```

Comment:
This test looks like it actually puts a lot of characters in there? (The fp.repeat call). Can you verify this does what we want it to?